layouts/shortcodes/imgproc.html

```diff
@@ -14,7 +14,7 @@
 
 {{ $image := .Scratch.Get "image" }}
 
-<figure>
+<figure class="{{ with .Get 3}}{{ . }}{{ else -}} left {{- end }}" >
```

Comment:
```suggestion
<figure class="{{ with .Get "position"}}{{ . }}{{ else -}} left {{- end }}" >
```

Reply:
I'd prefer to stick to the named params ;-)

Reply:
Ah, that's where things get a little tricky. We can't mix positional params with named params (throws an error), and the shortcode already uses positional params for the other data. However, it looks like we could use `.Scratch` to set our own local variable for it. I'm just unclear on how it works. I'll look into it, though.

Reply:
Right, I forgot the context and didn't look at the whole implementation ;-( If your changes work, I'm fine with local "named" param ;-D